client/src/mdn.scss

```diff
@@ -167,6 +168,23 @@ div.content {
       border-bottom: 1px solid #eaecef;
     }
   }
+
+  .banner {
+    padding: 1rem;
+    margin: 0.5rem;
+  }
+
+  .deprecated-banner {
+    background-color: #ffe7e8;
+  }
+
+  .experimental-banner {
+    background-color: #fff3d4;
+  }
+
+  .nonstandard-banner {
+    background-color: #ffe7e8;
+  }
```

Comment:
Eventually, this is not going to fly. Instead, we need to move this out of `mdn.scss` and put them into a `src/ingredients/status-banners.scss` file and in this .js file we do a `import "./status-banners.scss";`

But I don't think the story of we do CSS-modules or SCSS in stumptown-renderer has solidified yet. 
So, can you do me a favor and add a SCSS code comment for these 4 CSS selectors like `/* TODO: Move to status-banners.scss */`